Libs/MRML/Widgets/qMRMLSliceControllerWidget.cxx

```diff
@@ -409,6 +408,7 @@ void qMRMLSliceControllerWidgetPrivate::init()
     ctkDoubleSpinBox::DecimalsByShortcuts |
     ctkDoubleSpinBox::DecimalsByKey |
     ctkDoubleSpinBox::DecimalsAsMin );
+  this->SliceOffsetSlider->spinBox()->setStyleSheet("color: black; background-color: transparent;");  // Slicer light/dark mode does not manipulate Slice controller colors
```

Comment:
```suggestion
  // Slice controller background color is independent from the color palette, therefore the color of text and controls are hardcoded to black
  this->SliceOffsetSlider->spinBox()->setStyleSheet("color: black; background-color: transparent;");
```

Reply:
Thanks for the updated comment suggestion. I have proceeded with this version!